### PR TITLE
Create knex jobs table

### PR DESCRIPTION
**Pull request by eliasfang**
### Administrative Info
Monday Board ID: 853648344
Make sure your branch name conforms to: `<feature/staging/hotfix/...>/[username]/[Monday Item ID]-[3-4 word description separated by dashes]`. Otherwise, please rename your branch and create a new PR.

### Changes
What changes did you make?
- Created a new migration file with knex
- Created the 'jobs' table

### Testing
How did you confirm your changes worked? 
- Proofread code for errors
- Code does not display any errors in editor

### Confirmation of Change 
Upload a screenshot, if possible. Otherwise, please provide instructions on how to see the change.

| knex jobs table |
| --- |
| I cannot check for functionality yet since I need to get the code checked first. I will make sure to screenshot the actual changes for future PR's. |

### CODE REVIEW

**Comment by wes-chen** [comment]
Hey I know you wrote "self checked", but what exact steps did you take to self check and confirm your changes?

File: app/database/migrations/20201201151227_create-jobs-table.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+
+exports.up = function(knex) {
+    return knex.schema.createTable('jobs', tbl => {
+        tbl.increments('id').unique().nonNullable();
+        tbl.integer('class_id').unsigned().nonNullable();
+        tbl.timestamps(true, true); // creates 'created_at' and 'update_at' columns (useTimestamps = true, defaultToNow = true)
+        tbl.datetime('scheduled_time');
+        // TODO: @Arnav add status(enum)
```
Comment (mingkaili) [inline app/database/migrations/20201201151227_create-jobs-table.js]: You can take out this comment, Arnav won't be making his changes in this file, he will be making a separate migration file.

**Review by mingkaili — CHANGES_REQUESTED** [review]
Great Start! go ahead and write the steps you take for self check, and take off the comment for Arnav's part. 

Knex is basically converting sql queries into javascript, and when we need to make change to a table, we don't go back and change the query, instead, we make a new query to alter the table. Therefore, changing existing knex migration file will make things very messy, and that is why whenever we need to make new changes, we just make a new migration file.

Also: make sure you add at least two more reviewers.

File: app/database/migrations/20201201151227_create-jobs-table.js (hunk 1 above)
Comment (eliasfang) [reply app/database/migrations/20201201151227_create-jobs-table.js]: Sure thing!

**Review by wes-chen — DISMISSED** [review]
Hey, great work on fixing your testing section! However, your confirmation of change should not literally be a screenshot of your code. How about writing steps to confirm that your jobs table exists or having a screenshot proof of the jobs table existence after following aforementioned steps?

**Review by eliasfang — COMMENTED** [review]
Thanks for your feedback!

**Review by mingkaili — APPROVED** [review]
LGTM! make sure to squash and merge

**Review by shravan1k — APPROVED** [review]
LGTM

**Pull request merged**